### ISSUE TITLE
400 returned following steps provided

### ISSUE THREAD

**Issue by odonnell-anthony** [vs4vijay/newman-reporter-influxdb#18]
Influx 1.8.4
newman 5.2.2

Follow steps provided in repo, is not successful returned status code 400

```
newman run .\collection.json -r influxdb --reporter-influxdb-server localhost --reporter-influxdb-port 8086 --reporter-influxdb-name postman --reporter-influxdb-measurement testing  --reporter-influxdb-username admin --reporter-influxdb-password admin

[+] Starting collection: test 1618250141484-0.08094263785491762
[1] Running httpbin post
[+] Finished collection: test (1618250141484-0.08094263785491762)
[-] ERROR: while sending data to InfluxDB Request failed with status code 400
```

Can query influx from command line
`curl -G 'http://localhost:8086/query?pretty=true' --data-urlencode "db=postman" --data-urlencode "q=show measurements"
{
    "results": [
        {
            "statement_id": 0
        }
    ]
}`

**Comment by vs4vijay** [comment]
Hey @odonnell-anthony, Can you try passing "identifier" option and try? looks like due to recent changes, a mandatory field "identifier" is introduced. Could you try passing `--reporter-influxdb-identifier` and see if that works, I will change it to optional field.

Example: `--reporter-influxdb-identifier anythingUnique`

**Comment by odonnell-anthony** [comment]
Using `--reporter-influxdb-identifier test` on the end of the same command in my first comment returned the same error

**Comment by manjunathin** [comment]
--reporter-influxdb-identifier anythingUnique     seems to be working for me

**Comment by manjunathin** [comment]
@odonnell-anthony : Can you try with syntax ex:
newman run AAA -e BBB --ssl-client-cert SSSS.pem --insecure -r cli,junit,html,influxdb --reporter-influxdb-server localhost --reporter-influxdb-port 8086 --reporter-influxdb-name ZZZ --reporter-influxdb-identifier test --reporter-influxdb-measurement testcase

**Comment by manjunathin** [comment]
> Using `--reporter-influxdb-identifier test` on the end of the same command in my first comment returned the same error



> Hey @odonnell-anthony, Can you try passing "identifier" option and try? looks like due to recent changes, a mandatory field "identifier" is introduced. Could you try passing `--reporter-influxdb-identifier` and see if that works, I will change it to optional field.
> 
> Example: `--reporter-influxdb-identifier anythingUni

**Comment by manjunathin** [comment]
@vs4vijay :Can you make --reporter-influxdb-identifier optional?

**Comment by vs4vijay** [comment]
Hey @manjunathin - I have released a new version (2.0.4) with identifier related changes. Could you please test and let me know?

CC: @odonnell-anthony

**Comment by manjunathin** [comment]
@vs4vijay : Working as expected with v2.0.4..Thank You
